.github/scripts/trymerge.py

```diff
@@ -422,8 +424,8 @@ def main() -> None:
         gh_post_comment(org, project, args.pr_num, f"Can't merge closed PR #{args.pr_num}", dry_run=args.dry_run)
         return
 
-    if pr.is_cross_repo():
-        gh_post_comment(org, project, args.pr_num, "Cross-repo merges are not supported at the moment", dry_run=args.dry_run)
+    if pr.is_cross_repo() and pr.is_ghstack_pr():
```

Comment:
Are cross-repo ghstack merges even possible?

I don't even think you can make a ghstack PR without being on the direct remote

Reply:
I don't think they are, but there are no harm in checking it is not the case